lib/workers/repository/process/lookup/index.ts

```diff
@@ -459,7 +477,7 @@ export async function lookupUpdates(
           res.updates.push({
             isPinDigest: true,
             updateType: 'pinDigest',
-            newValue: compareValue,
+            newValue: config.currentValue,
```

Comment:
here too?

Reply:
newValue is the same as currentValue. There is no config.newValue